docs/content/rest-api.md

```diff
@@ -710,7 +710,7 @@ Content-Type: application/json
 
 ## Data API
 
-The Data API exposes endpoints for reading and writing documents in OPA. For an introduction to the different types of documents in OPA see [How Does OPA Work?](../#how-does-opa-work).
+The Data API exposes endpoints for reading and writing documents in OPA. For an introduction to the different types of documents in OPA see [How Does OPA Work?](../how-does-opa-work).
```

Comment:
Funny enough, this change regresses on the preview:

```
{15:34} ~ ❯ docker run --rm -it linkchecker/linkchecker https://deploy-preview-2156--openpolicyagent.netlify.com/

<snip>

URL        `../how-does-opa-work'
Name       `How Does OPA Work?'
Parent URL https://deploy-preview-2156--openpolicyagent.netlify.com/docs/edge/rest-api/, line 2893, col 144
Real URL   https://deploy-preview-2156--openpolicyagent.netlify.com/docs/edge/how-does-opa-work
Check time 3.287 seconds
Size       1KB
Result     Error: 404 Not Found

<snip>
```
I think the crux of the issue is that there is no "how does opa work" section anymore. This should probably just be pointing to `../` (which the invalid `#` was inadvertently doing for us 😆 )